src/scss/settings/_variables.scss

```diff
@@ -0,0 +1,74 @@
+//
+// FONTS
+//
+$font-stack-primary: "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+//
+// COLORS
+//
+$c-terrible-red: #ff0000;
```

Comment:
I like the clever sample colors but since these are used in other SCSS partials it will be annoying to update the colors when setting up a project. What do you think about doing something like what we do with the z-index where we define the color system at the top of the file and then below it we create new variables that get used in partials. That way color updates will all be done in the same file. If we go that route we start a new `src/scss/settings_colors.scss` file.

Reply:
Yeah, you're right. Doing a find & replace in a codebase you might not be familiar with wouldn't be fun. I love the idea of mapping up the named colors with their functions.

Reply:
Now if you grep for `nasty`, `terrible`, and `painful`, the only results are in `src/scss/settings/_colors.scss`. So much easier to swap out!